src/haddock/modules/analysis/alascan/scan.py

```diff
@@ -0,0 +1,581 @@
+"""alascan module."""
+import os
+from pathlib import Path
+import shutil
+
+import numpy as np
+import pandas as pd
+import io
+from contextlib import redirect_stdout
+
+from haddock import log
+from haddock.libs.libalign import get_atoms, load_coords
+from haddock.libs.libplots import make_alascan_plot
+from haddock.modules.analysis.caprieval.capri import CAPRI
+from haddock.clis import cli_score
+
+ATOMS_TO_BE_MUTATED = ['C', 'N', 'CA', 'O', 'CB']
+
+RES_CODES = dict([
+    ("CYS", "C"),
+    ("ASP", "D"),
+    ("SER", "S"),
+    ("GLN", "Q"),
+    ("LYS", "K"),
+    ("ILE", "I"),
+    ("PRO", "P"),
+    ("THR", "T"),
+    ("PHE", "F"),
+    ("ASN", "N"),
+    ("GLY", "G"),
+    ("HIS", "H"),
+    ("LEU", "L"),
+    ("ARG", "R"),
+    ("TRP", "W"),
+    ("ALA", "A"),
+    ("VAL", "V"),
+    ("GLU", "E"),
+    ("TYR", "Y"),
+    ("MET", "M"),
+    ("ALY", "K"),
+    ("ASH", "D"),
+    ("CFE", "C"),
+    ("CSP", "C"),
+    ("CYC", "C"),
+    ("CYF", "C"),
+    ("CYM", "C"),
+    ("DDZ", "A"),
+    ("GLH", "E"),
+    ("HLY", "P"),
+    ("HY3", "P"),
+    ("HYP", "P"),
+    ("M3L", "K"),
+    ("MLY", "K"),
+    ("MLZ", "K"),
+    ("MSE", "M"),
+    ("NEP", "H"),
+    ("PNS", "S"),
+    ("PTR", "Y"),
+    ("SEP", "S"),
+    ("TOP", "T"),
+    ("TYP", "Y"),
+    ("TYS", "Y"),
+    ])
+
+
+def get_index_list(nmodels, ncores):
```

Comment:
Looks like this function should be somewhere more general, some lib maybe?

Reply:
makes sense. same function exists in #685 so I will have to call it also there